apps/test/integration/levelSolutions/applab1/ec_goals.js

```diff
@@ -13,34 +13,34 @@ export default {
 
```

Comment:
This test was failing because Chrome includes the accessible `<title>` text in these droplet blocks when you ask for the text-content of the top-level SVG group, whereas Phantom was only including the visible text.

Fixed by making the selector more specific so it only grabs the subset of visible text that changes from block to block, and updating the expectation accordingly.